credentialMixin.js

```diff
@@ -97,6 +101,14 @@ export const credentialMixin = {
         acc[key] = value;
         return acc;
       }, {});
+    },
+    getIcon(type) {
```

Comment:
We should not be creating an icon in here, this information is specific to a top level product.